### ISSUE TITLE
Support for openwebrx

### ISSUE THREAD

**Issue by criede** [lazywalker/DigiSkimmer#17]
Hello,
as KiwiSDR is a spinoff of openwebrx, I tried DigiSkimmer with openwebrx using my HackRF One as receiver, but DigiSkimmer could not connect. The URLs are different. 
It would be very good if DigiSkimmer could also support the API of openwebrx. Yes, HackRF is only for one channel with only 8bit resolution, but that's one channel more that no channel ;-). Also, HackRF One can receive on 6m and above, while KiwiSDR is limited to 10m and below. 
Thanks in advance
73 Christian DB9CR

**Comment by lazywalker** [comment]
Yes, the API of openwebrx is quite different, it might difficult to do that with digiskimmer.
but i think the openwebrx has a background running mode, it can decode and upload spots to pskreporter as well

**Comment by doccodyblue** [comment]
It indeed has background decoding and reporting FT8, FT4, WSPR etc to pskreporter and wsprnet. Working really reliable.
So maybe this can be closed?
